camera-manager.js

```diff
@@ -1,11 +1,15 @@
 import * as THREE from 'three';
 import {getRenderer, camera/*, orbitControls*/} from './renderer.js';
 import * as notifications from './notifications.js';
+import metaversefile from 'metaversefile';
 
 const localVector = new THREE.Vector3();
 
 const cameraOffset = new THREE.Vector3();
 let cameraOffsetTargetZ = cameraOffset.z;
+let cameraOffsetZ = cameraOffset.z;
+const raycaster = new THREE.Raycaster();

```

Comment:
We should use our own physics raycasting solution, which is the actual physics.

I think the method used here will slow down the app like crazy...

Reply:
Ok, I will use physics for raycast in next PR